Neos.Media/Classes/Command/MediaCommandController.php

```diff
@@ -388,6 +410,136 @@ public function renderThumbnailsCommand(int $limit = null, bool $quiet = false)
         }
     }
 
+    /**
+     * List all configurations for your imageVariants.
+     *
+     * Doesn't matter if configured in 'Settings.Neos.Media.yaml' or already deleted from this configuration.
+     * This command will find every single one for you.
+     */
+    public function listVariantPresetsCommand(): void
+    {
+        $currentPresets = $this->imageVariantService->getAllPresetsByConfiguration();
+
+        if (empty($currentPresets)) {
+            $this->outputLine('There was <b>no configuration</b> for variant presets found...');
+        } else {
+            $outputPresets = [];
+            foreach ($currentPresets as $presetIdentifier => $presetVariantNames) {
+                foreach ($presetVariantNames as $variants) {
+                    $outputPresets[] = [$presetIdentifier, $variants];
+                }
+            }
+            $this->outputLine('There is a <b>configuration</b> with following <b>variant presets:</b>');
+            $this->output->outputTable($outputPresets, ['Configured Presets', 'Label of variant']);
```

Comment:
I would add `identifier` and `variantName` to the table headers as they are needed in the remove command and it is not obvious which is which. (Also relevant on line 459)